github-link.php

```diff
@@ -9,7 +9,8 @@
 Author URI: http://www.online1.hu/webdesign/
 Domain Path:       /languages
 Text Domain:       github-link
-GitHub Plugin URI: https://github.com/szepeviktor/github-link
+GitHub Plugin URI: https://github.com/jjrohrer/github-link
+Depends: github-updater
```

Comment:
Please remove this line.
I don't use GHU. Maybe others aren't using GHU...
Don't tell anyone!!